core/logs/src/main/scala/org/typelevel/otel4s/logs/Logger.scala

```diff
@@ -19,19 +19,22 @@ package logs
 
 import cats.Applicative
 import cats.Monad
-import org.typelevel.otel4s.meta.InstrumentMeta
+import cats.~>
 
 /** The entry point into a log pipeline.
   *
   * @note
   *   the `Logger` is intended to be used only for bridging logs from other log frameworks into OpenTelemetry and is
   *   '''NOT a replacement''' for logging API.
+  *
+  * @see
+  *   [[https://opentelemetry.io/docs/specs/otel/logs/api/#logger]]
   */
 sealed trait Logger[F[_], Ctx] {
 
   /** The instrument's metadata. Indicates whether instrumentation is enabled.
     */
-  def meta: InstrumentMeta.Dynamic[F]
+  def meta: Logger.Meta[F, Ctx]
```

Comment:
Should we apply the same approach to other instruments (metrics/tracers)?

> https://opentelemetry.io/docs/specs/otel/metrics/api/#enabled
> https://opentelemetry.io/docs/specs/otel/trace/api/#enabled
> There are currently no required parameters for this API. Parameters can be added in the future, therefore, the API MUST be structured in a way for parameters to be added.

Reply:
If so, I would like to squeeze it into the 0.14 release, since it will be a breaking-breaking change ☹️